packages/webview_flutter/webview_flutter_wkwebview/lib/src/v4/src/webkit_webview_controller.dart

```diff
@@ -132,10 +132,14 @@ class WebKitWebViewController extends PlatformWebViewController {
         NSObject object,
         Map<NSKeyValueChangeKey, Object?> change,
       ) {
-        if (weakReference.target?._onProgress != null) {
+        if (weakReference.target?._currentNavigationDelegate?._onProgress !=
+            null) {
+          final ProgressCallback progressCallback =
+              weakReference.target!._currentNavigationDelegate!._onProgress!;
```

Comment:
Nit: this can be adjusted slightly to eliminate the force-unwrap:

```
final ProgressCallback? progressCallback = weakReference.target?._currentNavigationDelegate?._onProgress;
if (progressCallback != null) {
  // now it's implicitly unwrapped and you can just use it.
}
```